tools/build-neatvnc.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
Replace this [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) with `#!/usr/bin/env bash`.